system-int-test/test/cmr/system_int_test/search/facets/collection_facets_v2_search_test.clj

```diff
@@ -67,6 +67,15 @@
    (let [query-params (merge search-params {:page-size 0 :include-facets "v2"})]
      (get-in (search/find-concepts-json :collection query-params) [:results :facets]))))
 
+(defn- search-and-return-v2-facets-errors
+  "Returns the facets returned by a search requesting v2 facets."
+  ([]
+   (search-and-return-v2-facets {}))
+  ([search-params]
+   (index/wait-until-indexed)
```

Comment:
Not a fan of throwing the `wait-until-indexed` call here. The CMR standard is to do this explicitly in tests.

Reply:
Could you please show me an example of exactly what you want it done?  I'm doing exactly the same as the function above.